docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -12,29 +12,27 @@ Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
 we recommend using ADLS Gen2. Learn more from `the official documentation
 <https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-overview>`_.
 
-Hive connector configuration
-----------------------------
+Hive connector configuration for Azure Storage credentials
+----------------------------------------------------------
 
-All configuration for the Azure storage driver is stored in the Hadoop
-``core-site.xml`` configuration file. The path to the file needs to be
-configured in the catalog properties file:
+To configure Trino to use Azure Storage credentials, set the following
+configuration properties in the catalog properties file. It is best to use this
+type of configuration if the primary storage account is linked to the cluster.
```

Comment:
It might be worth explaining that now core-site.xml file is needed in this case

Reply:
Let me know if this restructuring of information is more accurate.